events/13th-annual-users-meeting-2018.html

```diff
@@ -11,30 +11,191 @@
         <div class="row">        
             <div class="column">
                 <h2 class="event-day">Registration is now CLOSED!</h2>
+                <p class="event-day-description">Contact us on the mailing
+                lists if you want to attend; a small number of spaces are
+                available.</p>
             </div>
         </div>
         
         <hr class="whitespace">
         <div class="row">        
             <div class="column">
                 <h2 class="event-day">Day 1 — Wednesday 30th May, 2018 - Dalhousie Lecture Theatre</h2>
-                <p class="event-day-description">
-                    We will start with a morning session of short lightning
-                    talks from members of the community. Lightning talks set
-                    the scene for the community priorities and requirements
-                    that are the focus of the rest of the meeting. After
-                    lunch, we'll continue with any extra lightning talks—
-                    these have been very popular in the past, so please do
-                    sign up for one and use this opportunity to present your
-                    current work and your OME wishlist. These will be followed
-                    by an update on OME's progress and status and a few longer
-                    community talks. We'll finish the day with a poster
-                    session, drinks reception and dinner.
-                </p>
             </div>
         </div>
-        
-        
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">09:00 - 09:30</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Morning registration with tea/coffee</h3>
+            </div>
+        </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">09:30 - 09:45</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Deep Learning Object Segmentation using Omero, Orbit and Tensorflow</h3>
```

Comment:
OMERO (upper case). It may not match the title he supplied but is consistent with elsewhere on this page.